test/tools/auth0/handlers/themes.tests.js

```diff
@@ -0,0 +1,172 @@
+const { expect, assert } = require('chai');
+const { omit } = require('lodash');
+const {
+  default: ThemesHandler,
+  validTheme,
+} = require('../../../../src/tools/auth0/handlers/themes');
+
+function stub() {
```

Comment:
Created a minimal `stub` function to add more readability to all tests without including a new dependency like `sinon`.